internal/testserver/datastore/mysql.go

```diff
@@ -109,9 +110,9 @@ func (mb *mysqlTester) NewDatabase(t testing.TB) string {
 func (mb *mysqlTester) runMigrate(t testing.TB, dsn string) {
 	driver, err := migrations.NewMySQLDriverFromDSN(dsn, mb.options.Prefix)
 	require.NoError(t, err, "failed to create migration driver: %s", err)
-	err = migrations.Manager.Run(driver, migrate.Head, migrate.LiveRun)
+	err = migrations.Manager.Run(context.Background(), driver, migrate.Head, migrate.LiveRun)
```

Comment:
I think the duplication here might be in error

Reply:
Indeed, thanks ! This must have slipped through my attention while resolving the rebase conflicts ...